test/URIFilePathSpec.hs

```diff
@@ -61,8 +65,8 @@ filePathUriSpec = do
     theFilePath `shouldBe` (Uri "file:///./Functional.hs")
 
   it "converts a Windows file path to a URI" $ do
-    let theFilePath = platformAwareFilePathToUri windowsOS "c:./Functional.hs"
-    theFilePath `shouldBe` (Uri "file:///c%3A/./Functional.hs")
+    let theFilePath = platformAwareFilePathToUri windowsOS "c:/Functional.hs"
+    theFilePath `shouldBe` (Uri "file:///c:/Functional.hs")
```

Comment:
Colons do not need to be escaped, see https://blogs.msdn.microsoft.com/ie/2006/12/06/file-uris-in-windows/

Reply:
I don’t know why the test was adding a `/` implicitely. Afaict that shouldn’t happen.